numba-dppy/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2012, Anaconda, Inc.
```

Comment:
Should this not be our License file?

Reply:
This is license file from numba-dppy. I think it contains an error.

Reply:
Updated.